packages/salesforcedx-sobjects-faux-generator/test/unit/fauxClassGenerator.test.ts

```diff
@@ -49,6 +58,21 @@ describe('SObject faux class generator', () => {
     );
   });
 
+  it('Should generate a faux class with field inline comments', async () => {
+    const gen = getGenerator();
+    const classContent = gen.generateFauxClassText(customSObject);
+
+    let standardFieldComment = `    /* Please add a unique name${EOL}`;
+    standardFieldComment += `    */${EOL}`;
+    standardFieldComment += `    global String Name;${EOL}`;
+    expect(classContent).to.include(standardFieldComment);
+
+    let customFieldComment = `    /* User field API name${EOL}`;
+    customFieldComment += `    */${EOL}`;
+    customFieldComment += `    global String Field_API_Name__c;${EOL}`;
+    expect(classContent).to.include(customFieldComment);
+  });
+
```

Comment:
If my comment [here](https://github.com/forcedotcom/salesforcedx-vscode/pull/1890/files#r369757522) is true, maybe we should have a test for replacing the delimiters as well.

Reply:
Agreed, let me add one for that scenario. I might also just change that method to public and write a test only for that parsing piece.